include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -8,23 +8,23 @@ struct SPhysicalDeviceFeatures
 {
     /* Vulkan 1.0 Core  */
     bool robustBufferAccess = false;
-    //VkBool32    fullDrawIndexUint32;
+    bool fullDrawIndexUint32 = false;
     bool imageCubeArray = false;
-    //VkBool32    independentBlend;
+    bool independentBlend = false;
     bool geometryShader    = false;
-    //VkBool32    tessellationShader;
+    bool tessellationShader = false;
     //VkBool32    sampleRateShading;
     //VkBool32    dualSrcBlend;
```

Comment:
`dualSrcBlend` is core in desktop GL, no clue about GLES

`sampleRateShading` is also core in GL 4.4, and in GLES 3.2
https://www.khronos.org/registry/OpenGL-Refpages/es3/html/glMinSampleShading.xhtml

but no clue what extension enables it for ES 3.1